be/src/olap/rowset/segment_v2/inverted_index_writer.cpp

```diff
@@ -421,7 +424,6 @@ class InvertedIndexColumnWriterImpl : public InvertedIndexColumnWriter {
                     // if this array is null, we just ignore to write inverted index
                     RETURN_IF_ERROR(add_document());
                     _doc->clear();
-                    _CLDELETE(ts);
```

Comment:
why delete here will make memory leak?

Reply:
The field now takes ownership of the TokenStream, eliminating the need for manual deletion.